test-data/unit/check-python310.test

```diff
@@ -1369,6 +1369,35 @@ match m3:
         reveal_type(m3)  # N: Revealed type is "Tuple[Union[builtins.int, builtins.str]]"
 [builtins fixtures/tuple.pyi]
 
+[case testMatchEnumMultipleChoices]
```

Comment:
This case is not strictly needed for this change. It just demonstrates how this should work normally. But I would think this is already covered. What do you think?